docs/products/postgresql/concepts/upgrade-failover.rst

```diff
@@ -65,7 +65,7 @@ In case of failover or controlled switchover of an Aiven for PostgreSQL service,
 
 .. important::
 
-    The recreation of replication slots feature is enabled automatically.
+    The recreation of replication slots feature is enabled automatically and doesn't require restarting the nodes.
```

Comment:
I think the second part of the sentence is old, needs to be removed.

Reply:
@alemedeiros , @divdby0 explicitly requested it as is.